jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsAttribute.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2019, Red Hat Inc., and individual contributors as indicated
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.as.jaxrs;
+
+import org.jboss.as.controller.SimpleAttributeDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
+import org.jboss.as.controller.operations.validation.ModelTypeValidator;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+
+/**
+ * Jaxrs configuration attributes.
+ *
+ * @author <a href="mailto:rsigal@redhat.com">Ron Sigal</a>
+ */
+public interface  JaxrsAttribute {
+
+    public static final String RESTEASY_PARAMETER_GROUP = "resteasy-parameter-group";
+
+    SimpleAttributeDefinition JAXRS_2_0_REQUEST_MATCHING = new SimpleAttributeDefinitionBuilder(JaxrsConstants.JAXRS_2_0_REQUEST_MATCHING, ModelType.BOOLEAN)
+            .setRequired(false)
+            .setAllowExpression(true)
+            .setValidator(new ModelTypeValidator(ModelType.BOOLEAN, false))
+            .setDefaultValue(ModelNode.FALSE)
+            .setAttributeGroup(RESTEASY_PARAMETER_GROUP)
```

Comment:
Given the prefix of this attributes (`jaxrs-2`), does it really belong to the RESTEASY_PARAMETER_GROUP attribute group (which to me signals attributes that are specific to the RestEasy implementation)?

Reply:
Hi @jmesnil,

Sigh. I was about to say, it's a bad name, I have no idea where that came from. Then I did a little search and discovered that I named it myself for https://issues.redhat.com/browse/RESTEASY-1951. It's meant to provide backwards compatibility with the JAX-RS 2.0 request matching algorithm. So, it's about JAX-RS but it's specific to RESTEasy. I should have named it more conventionally.

-Ron